### DISCUSSION TITLE
Smartcard-support for the DIY

### DISCUSSION THREAD

**Discussion by k9ert** [cryptoadvance/specter-diy#113] [Ideas]
It would be cool to have smartcard-support for the DIY to have equal functionality then the Shield.
Stepan said on the chat after my question which smartcard-reader to buy: 

> We need to add USB host functionality in micropython 
> and implement a driver for the smartcard reader. They 
> all use a standard interface, so should be compatible with any reader.

So, researching a bit, is this piece of hardware maybe a way to make the development of it easier?
https://forum.micropython.org/viewtopic.php?t=6013

**Comment by sachaweb** [comment]
Hi. STM32F469I-DISCO developer board seems to be out of stock everywhere. Is there another component that can be used instead?

**Comment by k9ert** [comment]
Good question. But even Raspi-Zeros are out of stock for the seedsigner so i doubt that changing device would help here.
